README.md

```diff
@@ -104,4 +104,4 @@ Follow Milvus on social media:
 
 - [Medium](https://medium.com/@milvusio)
 
-- [Twitter](https://twitter.com/milvusio)https://twitter.com/milvusio)
```

Comment:
This link was extra So I removed it.

Reply:
- [Twitter](https://twitter.com/milvusio)